### PR TITLE
Add blog post accessibility checklist 

### PR DESCRIPTION
**Pull request by isabela-pf**
Hi there! I put together a short list of things to review when submitting a blog to avoid some simple accessibility problems. It's designed to be a PR template so that it can't be missed.

While this could be useful for blog posts at any time, I'd love to get this PR in before all the intern blog posts start rolling in because I think that would be a good opportunity to try out this system on a lot of different content types.

Thanks in advance for any review! 🌻

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--Hi there! If you aren't submitting a blog post, you can delete this message.
+
+Thank you for submitting a blog post! We want to make sure content produced for Quansight Labs is accessible, so please address the the following guidelines in your post.-->
+
+
+Text styling
+- [ ] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
+- [ ] If there are headers, they use the proper header tags in order to do so.
```
Comment (trallard) [inline .github/pull_request_template.md:8]: This has come and bitten a few folks - can we add an additional item or extend this to "There is only one level-one header"

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- [ ] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.
+
+Non-text content
+- [ ] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
```
Comment (trallard) [inline .github/pull_request_template.md:12]: what about links? - I have zero idea how to phrase it though

- **bad**: check [here](https://labs.quansight.org/)
- **good**: check the [Quansight labs website](https://labs.quansight.org/)

**Review by trallard — COMMENTED** [review]
Overall looks great  and I am sure this will help make our posts much more accessible moving forward

**Review by trallard — APPROVED** [review]
Added a couple minor edits - nitpicks really. This looks good to me for merge.

**Comment by melissawm** [comment]
@isabela-pf or @MarsBarLee it would be very helpful to have something like this added to the NumPy docs, maybe as a part of the [Reviewer Guidelines](https://numpy.org/devdocs/dev/reviewer_guidelines.html)?

**Comment by isabela-pf** [comment]
Yes, I think that could be helpful! The main question is if there are items that are needed for documentation that are missing here or things we'd want to remove, but I think that can be discussed elsewhere. 

I can open a draft PR based on this in the next week. Thanks for asking!

**Comment by MarsBarLee** [comment]
Yes, I agree with linking this checklist in the Reviewer Guidelines. Thanks for doing the PR Isabela!

**Pull request merged**